en/components/sprite.md

```diff
@@ -34,7 +34,7 @@ Currently, the Sprite component supports five rendering modes:
 
 - `Sliced mode`: the image is cut up into a 9-slicing and according to certain rules is scaled to fit freely set dimensions (`size`). It is usually used in UI elements or to make images that can be enlarged infinitely without influencing the image quality into images cut up into a grid to save game resource space. Please read [Use a Sliced Sprite to make an UI imagee](../ui/sliced-sprite.md) for details.
 
-- `Tiled mode`: The image will be repeated to fit the size of the Sprite. If the SpriteFrame is [9-sliced](../ui/sliced-sprite.md), the corners will also remain unscaled while the other areas will be repeated.
+- `Tiled mode`: The image will be repeated to fit the size of the Sprite. If the SpriteFrame is [9-sliced](../ui/sliced-sprite.md), then when the image is tiled, the edges will maintain the size of the four borders, and the central part is repeated.
```

Comment:
If xxx, then when xxx，感觉语法有问题。在英文里很少看到这种句式。参考一下机翻：
```
If the SpriteFrame contains a nine-panel configuration, then the edges of the image will remain nine-panel quadrangles when tiled, while the center will keep repeating.
```
the edges will maintain the size of four borders 也很奇怪，字面意思是边缘会保留四个边的尺寸？主语不对。
直接说四个边会保留尺寸就好了。four borders 也很奇怪，要嘛就 corners（某 U 的翻译），要嘛就 borders。
参考某引擎的说明：
```
The SpriteRenderer will render the sprite as a 9-slice image where the corners will remain constant and the other sections will tile.

In this draw mode, the corners will remain unscaled while the other sections will be repeated instead of stretched. It uses the Sprite.border value to determine how each part (border and center) should be tiled.
```